samples/addons/README.md

```diff
@@ -87,13 +87,13 @@ The [Prometheus Operator](https://github.com/coreos/prometheus-operator) manages
 As an alternative to the standard Prometheus deployment, we provide a `ServiceMonitor` to monitor the Istio control plane and `PodMonitor`
 Envoy proxies. To use these, make sure you have the Prometheus operator deployed, then run `kubectl apply -f samples/addons/extras/prometheus-operator.yaml`.
 
-> **Note**  
+> **Note**
```

Comment:
This will result in displaying everything in one line. I made this change on purpose. I wanted to make it more visible.

Reply:
Then, let's update as so. Is this what you wanted?